frame/support/src/traits/tokens/fungibles.rs

```diff
@@ -265,13 +265,7 @@ pub trait Create<AccountId>: Inspect<AccountId> {
 
 /// Trait for providing the ability to destroy existing fungible assets.
 pub trait Destroy<AccountId>: Inspect<AccountId> {
-	/// The witness data needed to destroy an asset.
-	type DestroyWitness;
-
-	/// Provide the appropriate witness data needed to destroy an asset.
-	fn get_destroy_witness(id: &Self::AssetId) -> Option<Self::DestroyWitness>;
-
-	/// Destroy an existing fungible asset.
+	/// Start the destruction an existing fungible asset.
 	/// * `id`: The `AssetId` to be destroyed.
 	/// * `witness`: Any witness data that needs to be provided to complete the operation
```

Comment:
no witness arg anymore

Reply:
Thanks for spotting this. I've taken it out.